southpark-search/tests/distributed/test_integration.sh

```diff
@@ -0,0 +1,34 @@
+set -e
+
+if [ "${PWD##*/}" != "southpark-search" ]
+  then
+    echo "test_integration.sh should only be run from the southpark-search base directory"
+    exit 1
+fi
+
+export $(grep -v '^#' tests/distributed/.env | xargs -d '\n')
```

Comment:
is this the line passing the `env variables`?

Reply:
no, It sets them locally for this shell context. added a comment.